.circleci/config.yml

```diff
@@ -44,72 +62,99 @@ jobs:
 
       - save_cache:
           paths:
-            - ./emsdk/emsdk
             - ~/.ccache
           key: v1-emsdk-{{ checksum "emsdk/Makefile" }}-v20201205-{{ .BuildNum }}
```

Comment:
It now doesn't make sense to use `emsdk/Makefile` as our cache key. It would make sense to use a new cache if we change emscripten version, so perhaps `Makefile.envs`?